t/01-sanity.t

```diff
@@ -34,6 +48,12 @@ __DATA__
             local l = require("resty.lmdb")
 
             ngx.say(l.set("test", "value"))
+            local file1=io.input("/tmp/test.mdb/data.mdb")  
+            local str=io.read("*a")
+            local _,q
+            _, q=string.find(str, 'test')
```

Comment:
also check for value; also please be consistent with the style in test as well, insert spaces before and after `=`